deployment/functionapp.bicep

```diff
@@ -2,30 +2,43 @@ var location = resourceGroup().location
 
 // Input params
 param funcAppStorageAccountName string
-param funcAppStorageConnectionStringSecretName string
 param funcAppStorageSkuName string
 param appInsightsName string
 param appServicePlanName string
 param functionAppName string
 param logAnalyticsName string
 param managedIdentityName string
-param keyVaultName string
-param storageConnectionStringSecretName string
-param tags object = {}
+param cosmosdbAccountName string
+param diAccountName string
+param openAIAccountName string
+param storageAccountName string
 
 
 // Get existing managed identity resource
-resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' existing= {
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-07-31-preview' existing= {
   name: managedIdentityName
 }
 
-// Get existing keyvault resource
-resource keyvault 'Microsoft.KeyVault/vaults@2023-02-01' existing = {
-  name: keyVaultName
+resource cosmosDB 'Microsoft.DocumentDB/databaseAccounts@2024-05-15' existing= {
+  name: cosmosdbAccountName
 }
 
+resource documentIntelligence 'Microsoft.CognitiveServices/accounts@2024-04-01-preview' existing = {
+  name: diAccountName
+}
+
+resource openAi 'Microsoft.CognitiveServices/accounts@2024-04-01-preview' existing = {
+  name: openAIAccountName
+}
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2023-05-01' existing= {
+  name: storageAccountName
+}
+var storageConnectionStringValue = 'DefaultEndpointsProtocol=https;AccountName=${storageAccount.name};EndpointSuffix=${environment().suffixes.storage};AccountKey=${storageAccount.listKeys().keys[0].value}'
```

Comment:
We can't get away from using the connection string here?

Reply:
I am trying out to completely remove connection string but not able to directly hookup blob trigger with managed identity so took this approach rightnow.

Reply:
Totally understand.

Reply:
Let me try to remove it in follow-up PR.